detection-rules/impersonation_human_resources.yml

```
@@ -62,11 +60,26 @@ source: |
     )
   )
   // Request and Urgency
-  and any(ml.nlu_classifier(body.current_thread.text).entities,
-          .name == "request"
-  )
-  and any(ml.nlu_classifier(body.current_thread.text).entities,
-          .name in ("urgency", "financial")
+  and (
+    (
+      (length(body.current_thread.text) > 100)
```

shouldn't need the outer `(` and `)` around this
```suggestion
      length(body.current_thread.text) > 100
```